zerver/tests/test_openapi.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Any
+
+from zerver.lib.test_classes import ZulipTestCase
+from zerver.lib.openapi import (
+    get_openapi_fixture, get_openapi_parameters,
+    matches_openapi_schema, to_python_type, SchemaError
+)
+
+TEST_ENDPOINT = '/messages/{message_id}'
+TEST_METHOD = 'patch'
+TEST_RESPONSE = '400'
+
+
+class OpenAPITest(ZulipTestCase):
```

Comment:
What is this test class testing?  It feels like it's testing our OpenAPI parsing? 

Might be worth a docstring, and renaming it to `OpenAPIParsingTest`.

Reply:
I decided to rename it to `OpenAPIToolsTest`, since `zerver/lib/openapi.py` isn't exactly for parsing our OpenAPI content, but manage the stuff that has already been parsed (i.e. tools that help us handle OpenAPI stuff).

Let me know if you still prefer `OpenAPIParsingTest`.

Reply:
that's fine.